sdk/core/azure-core/src/http/request.cpp

```diff
@@ -71,3 +76,27 @@ std::map<std::string, std::string> Request::GetHeaders() const
 }
 
 BodyStream* Request::GetBodyStream() { return m_bodyStream; }
+
+std::vector<uint8_t> const& Request::GetBodyBuffer() { return m_bodyBuffer; }
```

Comment:
should be const?

Reply:
suggestion

Reply:
What if the body buffer is kind of big and we want to move it?